WebContent/WEB-INF/jsp/login.jsp

```diff
@@ -81,7 +81,7 @@
 
 
               <div class="login-button" align="center">
-                <input type="submit" value="<fmt:message key="login.loginButton"/>"  style="padding: 5px 20px; margin-left:30%; margin-top:10px;"/>
+                <input id="login" type="submit" value="<fmt:message key="login.loginButton"/>"  style="padding: 5px 20px; margin-left:30%; margin-top:10px;"/>
```

Comment:
Why do we add id=login

Reply:
This property has been added for automation tests.That parameter is standard of input. We can use in automation property name also  , but for now it is now important.